### PR TITLE
Add slipit tool : Utility for creating ZipSlip archives

### PR DESCRIPTION
**Pull request by Fisjkars**
Add slipit tool : Utility for creating ZipSlip archives

### CODE REVIEW

File: sources/install.sh [hunk 1]
```diff
@@ -1114,6 +1114,12 @@ function install_ysoserial() {
   add-test-command "ysoserial --help; ysoserial --help |& grep 'spring-core:4.1.4.RELEASE'"
 }
 
+function install_slipit() {
+  # Utility for creating ZipSlip archives
+  colorecho "Installing slipit"
+  pip3 install slipit
```
Comment (ShutdownRepo) [inline sources/install.sh:1120]: Can you switch to `python3 -m pip install slipit`?
Can you also add `add-test-command`, and if you have some example commands `add-history` as well

**Review by ShutdownRepo — CHANGES_REQUESTED** [review]

File: sources/install.sh (hunk 1 above)
Comment (QU35T-code) [reply sources/install.sh]: This tool has setup.py file. Please, can you switch pip install with pipx install ?

python3 -m pipx install slipit

Comment (Fisjkars) [reply sources/install.sh]: Hello, i've update the brach according to your reviews

**Review by ShutdownRepo — APPROVED** [review]

**Review by QU35T-code — APPROVED** [review]

**Review by QU35T-code — APPROVED** [review]

**Comment by ShutdownRepo** [comment]
A refactor is in progress ( https://github.com/ThePorgs/Exegol-images/pull/137 ) and should be over with shortly. Putting this PR on hold until then.
Once the refactor is over with, we will be able to update this PR and merge it 👍
Sorry for the inconvenience

**Comment by ShutdownRepo** [comment]
@Fisjkars step 1 of refactor is done, can you change your PR according to the new structure? 
1. sync the latest changed on our `dev` branch to your branch
2. move your changes where they're supposed to end up at

If that's easier for you, you can close this PR and create another one if you'd like

**Pull request closed without merging**